dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -107,8 +109,74 @@
 
     private transient ContentletAPI contentletAPI;
     private transient UserAPI userAPI;
+	private transient WriteRequest.RefreshPolicy refreshPolicy;
+
+	public Object getRefreshPolicy () {
+
+		return refreshPolicy;
+	}
+
+	/**
+	 * This method is used more a low-level library see these methods to assign the refresh policy
+	 * {@link #waitUntilContentRefresh()}
+	 * {@link #immediateContentRefresh()}
+	 *
+	 * @param refreshPolicy
+	 */
+	public void setRefreshPolicy (final Object refreshPolicy) {
```

Comment:
`setIndexPolicy(final IndexPolicy)` and get rid of the null checks.

Reply:
ok